packages/backend/src/infrastructure/express/router/dashboard.routes.ts

```diff
@@ -0,0 +1,25 @@
+import express from 'express'
+import { CommonRoutesConfig } from './common.routes.config'
+import dashboardController from '../../../modules/dashboard/dashboard.controller'
+
+const V1 = '/v1'
```

Comment:
I wrote this on some other review, so please add comment:
`// TODO change to value from config`